src/arraytypes/arraytypes.jl

```diff
@@ -200,6 +200,7 @@ end
 
 Base.size(p::ValidityBitmap) = (p.ℓ,)
 nullcount(x::ValidityBitmap) = x.nc
+Base.all(x::ValidityBitmap) = x.nc == 0
```

Comment:
I don't love overloading `all` here; can we just call this `allvalid`? I don't think we need to overload anything for this.